build.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/sh
+
+fail() {
+    if [ "$NODE_DTRACE_PROVIDER_REQUIRE" == "" ]; then
+        echo "-----------------------------------------------------------------"
+        echo "Building dtrace-provider failed with exit code $1, falling back"
+        echo "on stub implementation."
+        echo ""
+        echo "Re-run install with V set in your environment to see the build"
+        echo "output, or NODE_DTRACE_PROVIDER_REQUIRE=hard to force an"
+        echo "installation failure."
+        echo "-----------------------------------------------------------------"
+    fi
+
+    if [ "$NODE_DTRACE_PROVIDER_REQUIRE" == "hard" ]; then
+        exit 1
+    else
+        exit 0
+    fi
+}
+
+buildUSDT() {
+    if [ "$NODE_DTRACE_PROVIDER_REQUIRE" == "" ] && [ "$V" == "" ]; then
+        exec 1> /dev/null
```

Comment:
Does build.sh run in its own sub-process? Otherwise, would this potentially affect where the output for other modules' installation goes?

Reply:
Both of these functions run in their own subprocess so that we can discard the output from the command and still be able to print out the failure banner message.

Reply:
Ah right, I had missed the parentheses around the calls to those two functions. Sounds good to me!